src/go/rpk/pkg/cli/cmd/cloud/clusters_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package cloud_test
+
+import (
+	"bytes"
+	"errors"
+	"strings"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/cloud"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/vcloud/yak"
+)
+
+func TestClusters(t *testing.T) {
+	tests := []struct {
+		name           string
+		client         yak.CloudApiClient
+		expectedOutput []string
+		expectedError  string
+	}{
+		{"success", &mockYakClient{}, []string{"notready", "false"}, ""},
```

Comment:
Can you reformat this and other test cases I might have missed, so each field goes in a new line? Long lines make it harder to read 🙏🏽

Reply:
🤦‍♀️ I wanted to do that but lost track while doing it. sorry